mocks/SSMAPI.go

```diff
@@ -0,0 +1,9688 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Having a 10k lines of generated mock code doesn't seem that necessary here.
As only one method from the interface is used, I'd suggest something like this (defined locally to the ssm backend):

```
type fakeSSM struct {
         ssmiface.SSMAPI
         getParams func(aws.Context, *ssm.GetParametersByPathInput, ...request.Option) (*ssm.GetParametersByPathOutput, error)
}

func (s fakeSSM) GetParametersByPathWithContext(ctx aws.Context, inp *ssm.GetParametersByPathInput, opts ...request.Option) (*ssm.GetParametersByPathOutput, error) {
	return s.getParams(ctx, inp, opts)
}
```

Then you can use the fake type with closures of your own choice to check the results appropriately, and no need for the generated mock stuff.

Reply:
Removed the mock, and, just brought over the method under test.

Reply:
Also removed mockery from `go.mod`.